runtime/expr/function/ztests/regexp-replace.yaml

```diff
@@ -1,8 +1,9 @@
 zed: "yield regexp_replace(in, re, new)"
 
 input: |
-  {in:"-ab-axxb-",re:"a(x*)b",new:"T"}
+  {in:"-ab-axxb-",re:"ax*b",new:"T"}
   {in:"-ab-axxb-",re:"a(x*)b",new:"$1"}
+  {in:"-ab-axxb-",re:"a(?P<ZeroOrMoreX>x*)b",new:"foo$ZeroOrMoreX bar"}
```

Comment:
The space char included in ` bar` here is intentional. When #4444 is addressed I'd propose adding another test that uses curly braces so the space char can be dropped.